bindings_ffi/src/mls.rs

```diff
@@ -329,6 +329,24 @@ impl FfiXmtpClient {
         Ok(state.into())
     }
 
+    /**
+     * Get the inbox state for each `inbox_id`.
+     *
+     * If `refresh_from_network` is true, the client will go to the network first to refresh the state.
+     * Otherwise, the state will be read from the local database.
+     */
+    pub async fn addresses_from_inbox_id(
```

Comment:
i think this comment was copied from `inbox_state`

Reply:
Yeah i modified the comment a little bit to refer to all addresses but otherwise its the same functions